src/data/constData.js

```diff
@@ -0,0 +1,132 @@
+export const CART_STATUS = {
+    itemsCount: 0,
+    itemsMRP: 0,
+    itemsAmount: 0
+}
+
+export const LOCATION = [
+    {
+        "name": "Bangaluru, India",
+        "value": "bangaluru"
+    },
+    {
+        "name": "Delhi, India",
+        "value": "delhi"
+    },
+    {
+        "name": "Bombay, India",
+        "value": "bomay"
+    }, 
+    {
+        "name": "Ahmedabad, India",
+        "value": "ahmedabad"
+    },
+    {
+        "name": "Kolkata, India",
+        "value": "kolkata"
+    }
+]
+
+export const SUB_HEADER_CATEGORIES = ['Vegetables & Fruits', 'Dairy & Breakfast', 'Muchies', 'Cold Drinks & Juices', 'Instant & Frozen Food',
+    'Tea, Coffee & Health Drinks', 'Bakery & Biscuits'];
+
+export const MORE_SUB_HEADER_CATEGORIES = [{
+    name: 'More',
+    value: null
```

Comment:
Why do we need to give the value as null? Why can't we remove it.

Reply:
Because otherwise it shows up on dropdown menu, and we don't want it to show up